DependencyInjection/Configuration.php

```diff
@@ -83,6 +83,13 @@ public function getConfigTreeBuilder()
                                 ->booleanNode('delete')->defaultValue(false)->end()
                             ->end()
                         ->end()
+                        ->arrayNode('orm')
```

Comment:
when changing the Configuration.php you also need to adjust Resources/config/schema/create-1.0.xsd